app/models/access_control_step.rb

```diff
@@ -94,13 +94,22 @@ def execute context
       media_object.governing_policies.delete( lease )
       lease.destroy
     end
+    #if context['add_lending_period_days'].present? || context['add_lending_period_hours'].present?
+    #  lending_period = build_lending_period(context['add_lending_period_days'], context['add_lending_period_hours'])
+    #  media_object.lending_period = lending_period if lending_period.positive?
+    #end
+
```

Comment:
Should this commented out code be removed?

Reply:
Yes, good catch.